docs/podman.1.md

```diff
@@ -182,13 +182,15 @@ the exit codes follow the `chroot` standard, see below:
 
 ## FILES
 
-**libpod.conf** (`/etc/containers/libpod.conf`)
+**libpod.conf** (`/usr/share/containers/libpod.conf`)
```

Comment:
Just to be clear, if you have podman installed, you *must* have a copy of that file /usr/share/containers/libpod.conf? I can see it as part of the Fedora package so I'm assuming it must exist.

Reply:
If no configuration file can be found we use compiled-in defaults

Reply:
Added
`Podman will use builtin defaults if no libpod.conf file is found.`